### DISCUSSION TITLE
How do I search files is deep sub-dirs ?

### DISCUSSION THREAD

**Discussion by JeviScript** [sayanarijit/xplr#553] [Q&A] [answered]
I have a fairly big project. I'd like to be able to find a file by it's name and explore other close files to it. The file might be nested quite deep in the project. Is there a way to do so using xplr ? I tried attaching fzf and find plugins but those return results only from the directory my xplr currently is.

**Comment by sayanarijit** [comment, marked as answer]
You might want to try something like this: https://github.com/sayanarijit/.files/blob/03197362286c077b1e6966c48151a1e39cec95b4/nixos/files/xplr/init.lua#L241

UPDATE: The latest version of [fzf.xplr](https://github.com/sayanarijit/fzf.xplr) provides an option `recursive` for the same.

**Comment by sayanarijit** [comment]
UPDATE: The latest version of [fzf.xplr](https://github.com/sayanarijit/fzf.xplr) provides an option `recursive` for the same.
